.github/workflows/ci.yml

```diff
@@ -25,5 +26,9 @@ jobs:
         run: export
       - name: Compile
         run: make
-      - name: Test
+      - name: Test - Linux
+        if: matrix.os == 'ubuntu-18.04'
```

Comment:
I can see what you're doing here; but it means we end up with a "Test - mac" build step on Linux, and a "Test - Linux" build step on Mac. 

My preference here would be to either include this a part of the run script (e.g., set the library suffix as a shell-level `if` that is part of the "test" step; or, worst case, add two separate job definitions, one for Linux and one for Mac. 

The last one would inevitably involve some config duplication - maybe that could be avoided with YAML snippet inclusion, but honestly, the definition is sufficiently straightforward that I'm honestly not overly bothered by the copy-paste nature of the approach.

Reply:
+1